aspnetcore/razor-components/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Introduction to Razor Components
+author: guardrex
+description: Explore Blazor, a new experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
```

Comment:
No need to state experimental status here, since the doc uses the include that states it.

```suggestion
description: Explore Blazor, a .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
```